scripts/processing/Makefile

```diff
@@ -11,8 +11,13 @@ help:
 check:
 	$(PYTHON) check_system.py
 
-fetch-recon:
+clean:
+	rm -rf ../../ds119
```

Comment:
ds119? did you see the `clean` directive in this PR: https://github.com/mne-tools/mne-biomag-group-demo/pull/25/files?